configs/forcedPhotCcd.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+from lsst.utils import getPackageDir
+
+subaruConfig = os.path.join(getPackageDir("obs_subaru"), "config", "forcedPhotCcd.py")
+if os.path.exists(subaruConfig):
+    config.load(subaruConfig)
+hscConfig = os.path.join(getPackageDir("obs_subaru"), "config", "hsc", "forcedPhotCcd.py")
+if os.path.exists(hscConfig):
+    config.load(hscConfig)
```

Comment:
In the current `ci_hsc`  the config override from https://github.com/lsst/ci_hsc/blob/master/forcedPhotCcdConfig.py
has `config.references.skipMissing = True`. Do we need that added here?

Reply:
I think that does not apply to us, as we are using the preflight to decide on what to run on.